.storybook/components/DesignTokens/Tier1/TypographyPresets.js

```diff
@@ -1,161 +1,60 @@
-      <Grid>
-        <TokenSpecimen
-          name="eds-typography-preset-001"
-          specimenClassName={styles['u-typography-preset-001']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-001-bold"
-          specimenClassName={styles['u-typography-preset-001-bold']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-002"
-          specimenClassName={styles['u-typography-preset-002']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-002-bold"
-          specimenClassName={styles['u-typography-preset-002-bold']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-003"
-          specimenClassName={styles['u-typography-preset-003']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-003-bold"
-          specimenClassName={styles['u-typography-preset-003-bold']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-004"
-          specimenClassName={styles['u-typography-preset-004']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-004-bold"
-          specimenClassName={styles['u-typography-preset-004-bold']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-005"
-          specimenClassName={styles['u-typography-preset-005']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-005-light"
-          specimenClassName={styles['u-typography-preset-005-light']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-005-bold"
-          specimenClassName={styles['u-typography-preset-005-bold']}
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-006"
-          specimenClassName={styles['u-typography-preset-006']}
-          variant="typography-body"
-        />
-
+    const renderTypeToken = (size, value) => {
+      return (
         <TokenSpecimen
-          name="eds-typography-preset-006-bold"
-          specimenClassName={styles['u-typography-preset-006-bold']}
+          comment="font size / line height (px)"
+          name={`eds-typography-preset-${size}`}
+          specimenClassName={`u-typography-preset-${size}`}
+          value={value}
           variant="typography-body"
         />
+      );
+    };
 
-        <TokenSpecimen
-          name="eds-typography-preset-007"
-          specimenClassName={styles['u-typography-preset-007']}
-          variant="typography-body"
-        />
+    return (
+      <Grid>
+        {renderTypeToken('001', '40 / 48')}
```

Comment:
I really like that this exposes the underlying values so it's easier to compare to a design or an existing feature and make sure you're using the right typography level, but I'm also wondering if this is going to be hard to keep in sync with the tokens if we ever make changes to the actual typography styles 🤔 No suggestion; just putting it out there.

Reply:
yeah, great point 😬  let me see if we can grab the font size token values, but line height we're not exposing as tokens at all currently